---
 setup.py |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- thefuck.orig/setup.py
+++ thefuck/setup.py
@@ -50,6 +50,6 @@ setup(name='thefuck',
       install_requires=install_requires,
       extras_require=extras_require,
       entry_points={'console_scripts': [
-          'thefuck = thefuck.main:main',
+          'thefuck.real = thefuck.main:main',
           'thefuck-alias = thefuck.main:print_alias',
           'fuck = thefuck.main:how_to_configure_alias']})
