---
 thefuck/conf.py |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- thefuck.orig/thefuck/conf.py
+++ thefuck/thefuck/conf.py
@@ -26,7 +26,7 @@ ENV_TO_ATTR = {'THEFUCK_RULES': 'rules',
                'THEFUCK_PRIORITY': 'priority',
                'THEFUCK_DEBUG': 'debug'}
 
-SETTINGS_HEADER = u"""# ~/.thefuck/settings.py: The Fuck settings file
+SETTINGS_HEADER = u"""# ~/.config/thefuck/settings.py: The Fuck settings file
 #
 # The rules are defined as in the example bellow:
 #
@@ -73,7 +73,7 @@ class Settings(dict):
 
     def _setup_user_dir(self):
         """Returns user config dir, create it when it doesn't exist."""
-        user_dir = Path(os.path.expanduser('~/.thefuck'))
+        user_dir = Path(os.path.expanduser('~/.config/thefuck'))
         rules_dir = user_dir.joinpath('rules')
         if not rules_dir.is_dir():
             rules_dir.mkdir(parents=True)
